src/Features/CSharp/Portable/UseIsNullCheck/CSharpUseIsNullCheckCodeFixProvider.cs

```diff
@@ -17,12 +17,16 @@ protected override string GetIsNullTitle()
         protected override string GetIsNotNullTitle()
             => GetIsNullTitle();
```

Comment:
the feature is literall the "Use 'is null'" feature :)   You're now using "== null".

Reply:
Yes. But I thought we agreed on [that](https://github.com/dotnet/roslyn/pull/24173#issuecomment-357335703). Did I got you wrong?